### PR TITLE
Introduce index.ftl into base account theme

### LINKED ISSUE #26487: Update base theme to contain index.ftl with info message
There is confusion around extending a `base` theme - following the documentation leads to `InternalServer` as described in #21654. This task is to provide a better error reporting and updating the documentation so that it clearly states that `base` theme only contains translations (since removal of the original `account` v1 theme), and it is necessary to either provide own `index.ftl` when extending `base` theme, or extend some other theme which contains `index.ftl` file.

### PR DESCRIPTION
**Pull request by hmlnarik**
Fixes: #26487

### CODE REVIEW

File: docs/documentation/server_development/topics/themes.adoc [hunk 1]
```diff
@@ -59,9 +59,8 @@ A theme consists of:
 * Scripts
 * Theme properties
 
-Unless you plan to replace every single page you should extend another theme. Most likely you will want to extend the {project_name} theme, but you could also
-consider extending the base theme if you are significantly changing the look and feel of the pages. The base theme primarily consists of HTML templates and
-message bundles, while the {project_name} theme primarily contains images and stylesheets.
+Unless you plan to replace every single page you should extend another theme. Most likely you will want to extend the some existing theme. Alternatively, if you intend to provide your own implementation of the admin or account console,
+consider extending the `base` theme. The `base` theme consists of a message bundle and therefore such implementation needs to start from scratch, including implementation of the main `index.ftl` Freemarker template, but it can leverage existing translations from the message bundle.
```
Comment (hmlnarik) [inline docs/documentation/server_development/topics/themes.adoc:63]: This intentionally does not mention UiPageProvider SPI for extending admin console as that should be documented on its own.

File: themes/src/main/resources/theme/base/account/index.ftl [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```
Comment (jonkoops) [inline themes/src/main/resources/theme/base/account/index.ftl:1]: No need to capitalize this.
```suggestion
<!doctype html>
```

File: themes/src/main/resources/theme/base/account/index.ftl [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+<html>
```
Comment (jonkoops) [inline themes/src/main/resources/theme/base/account/index.ftl:2]: Ideally a `lang` attribute is added as well. Not sure if we already have a theme property for that.
```suggestion
<html lang="en">
```

File: themes/src/main/resources/theme/base/account/index.ftl [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    <head>
+        <title>${msg("accountManagementTitle")}</title>
+
+        <meta charset="UTF-8">
```
Comment (jonkoops) [inline themes/src/main/resources/theme/base/account/index.ftl:6]: Can be lowercase as well.
```suggestion
        <meta charset="utf-8">
```

File: themes/src/main/resources/theme/base/account/index.ftl [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
```
Comment (jonkoops) [inline themes/src/main/resources/theme/base/account/index.ftl:7]: No need for this, only old browsers use this, which we don't support.
```suggestion
        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
```

File: themes/src/main/resources/theme/base/account/index.ftl [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+        <meta name="robots" content="noindex, nofollow">
+
+        <!-- Inline favicon.ico for simplicity -->
+        <link type="image/x-icon" rel="icon" href="data:image/x-icon;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAACOklEQVQ4ja2TPUwTcRjGf/e/O+hB21BahFYBB6oBY0wIRnfnIxoHFYPG6KaDu3Fw0UQnY2KYTNFEFkxMbmAwDA4ODGI0okgNtSGtRaBUr+19tHfnQDCYGB3wmd4n75P3K88Lu4S0k+i6fh049Rf9F8MwLv6xgK7r+4G3QPQfTS8ZhpHZJmJH4hYQrYfCfOo/wvLACADfUwOUT5zF6Tu4rbuv63rHbxNkMplrtm0/UBSFSaudOUfBHRzhYX6Gie6jtHT10CrLnPkwQyzUgu/7n1VVLf4qMD8/byYSifBs7iuX5vJEknsZ6utlqrfJo03B3Q2Fw2GVk611TmsWrusiy/LWCtPT05OxWCwMMLFcJhJqxezq416Ph+u6nG+zSAifRrPJc6eNXEOgqiqStHU+kU6nxyRJ4snHFebWTMzUAOe0Oh2OiWmauK7L7U6LBctHCXzurMs4joNlWdi2jdgObNsmmj7EgT2dJEMtmKZJs9lElmUCs4KXW+R9tYEiBEIIPM/D931ENpt9DDCaitIdbeeb0+Sl10YZFU3TqFar3HhXJNYZpxZIXJXXsG2bcrlMrVZDjI+PXy4UCpVIJMLNeIOKB5uux5SSBODZqkXRE2xGu7gSMtmnBgRBAICiKMgAw8PDG7lcblSsFViXNRYlDdQQ3a9f8LRkUesfQlEVxvKvWF3JUyqVloB8qVTK7nTiJHChHomzMnicaMMm+WaWcixFcfAYyfwC8cISwA+g3zCMyn+x8q6fadf4CaUa9pUKnnXCAAAAAElFTkSuQmCC"/>
```
Comment (jonkoops) [inline themes/src/main/resources/theme/base/account/index.ftl:11]: We can also just omit the favicon here.

File: themes/src/main/resources/theme/base/account/index.ftl [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+    </head>
+
+    <body>
+        <h1>${msg("accountManagementWelcomeMessage")}</h1>
+        <div>${msg("accountManagementBaseThemeCannotBeUsedDirectly")}</div>
```
Comment (jonkoops) [inline themes/src/main/resources/theme/base/account/index.ftl:16]: Semantically this should probably be a paragraph.
```suggestion
        <p>${msg("accountManagementBaseThemeCannotBeUsedDirectly")}</p>
```

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Review by jonkoops — CHANGES_REQUESTED** [review]
One small note, otherwise LGTM

**Review by jonkoops — APPROVED** [review]
LGTM

**Review by ssilvert — APPROVED** [review]

**Pull request merged**